### DISCUSSION TITLE
after i uploaded the code mentioned correctly i get this error in the terminal while building the code.

### DISCUSSION THREAD

**Discussion by ratheesh1236** [Mair/esp32-course#130] [Course Qeustions]
assert failed: xEventGroupWaitBits event_groups.c:371 (uxBitsToWaitFor != 0)

Backtrace:0x400819f2:0x3ffc4c100x400883b9:0x3ffc4c30 0x4008ef25:0x3ffc4c50 0x4008b232:0x3ffc4d70 0x400d5e89:0x3ffc4da0 0x400d5c0c:0x3ffc4e50 0x4008b545:0x3ffc4e70 
0x400819f2: panic_abort at C:/ESP_IDF/esp-idf/components/esp_system/panic.c:402

0x400883b9: esp_system_abort at C:/ESP_IDF/esp-idf/components/esp_system/esp_system.c:128

0x4008ef25: __assert_func at C:/ESP_IDF/esp-idf/components/newlib/assert.c:85

0x4008b232: xEventGroupWaitBits at C:/ESP_IDF/esp-idf/components/freertos/event_groups.c:371 (discriminator 1)

0x400d5e89: wifi_connect_sta at C:\ESP_IDF_PROJECTS\wifi_connect_1\build/../components/wifi/connect.c:68

0x400d5c0c: wifi_connect at C:\ESP_IDF_PROJECTS\wifi_connect_1\build/../main/main.c:14

0x4008b545: vPortTaskWrapper at C:/ESP_IDF/esp-idf/components/freertos/port/xtensa/port.c:131

**Comment by Mair** [comment]
hi @ratheesh1236 

Can you please try comparing your code to the git repo. failing that please supply the video link and your code that you are having trouble with.
Thanks
